bin/run-migration-job.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+This script will either migrate or rollback the swoop database to a specified
+migration version. The migration version, action type (of either migrate/rollback),
+and an override parameter are all specified in the values.yml from the Swoop-DB helm
+chart.
+
+It requires the following environment variables be set:
+
+    - PGHOST:
+          Postgres K8s service name which will be used to connect to the database
+    - PGUSER:
+          name of the user (role) that will perform the migrations
+    - PGPORT:
+          port number of the database container
+    - PGDATABASE: name of the database which will be used for migrations
+    - Any additional libpq-supported connection parameters
+          (https://www.postgresql.org/docs/current/libpq-envars.html)
+"""
+import argparse
+import asyncio
+import os
+import sys
+
+from buildpg import V, funcs, render
+
+from swoop.db import SwoopDB
+
+APPLICATION_ROLES = ["user_api", "user_caboose", "user_conductor"]
+
+
+def stderr(*args, **kwargs) -> None:
+    kwargs["file"] = sys.stderr
+    print(*args, **kwargs)
+
+
+def check_positive(value) -> int:
+    ivalue = int(value)
+    if ivalue <= 0:
+        raise argparse.ArgumentTypeError(
+            "%s is an invalid migration version number. Only \
+                positive values are supported."
+            % value
+        )
+    return ivalue
+
+
+async def run_migrations() -> None:
+    dbname: str = os.environ["PGDATABASE"]
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--action",
+        choices=["migrate", "rollback"],
+        help="An action to take, either migrate or rollback",
+    )
+    parser.add_argument(
+        "--version",
+        type=check_positive,
+        help="The migration version to which to migrate/rollback the database",
+    )
+
+    parser.add_argument(
+        "--wait_override",
+        choices=["true", "false"],
+        help="Override option to skip waiting for active connections to close. \
+        If specified, it is true, and if not specified it is false.",
+    )
```

Comment:
Using choices for a bool parameter is rather strange to me. Doesn't it require the user actually specify `true` or `false`? As an alternative, see `store_true`: https://docs.python.org/3/library/argparse.html#action.

Also, I believe the help message has a bunch of spaces in it (`\` only escapes the line break).

How about:

```suggestion
    parser.add_argument(
        "--no-wait",
        action="store_true",
        help="Skip waiting for active connections to close. Default false.",
    )
```